README.adoc

```diff
@@ -66,59 +66,59 @@ See also: <<your-profile-disturbs-me>>.
 [[why-end-dictatorship]]
 === Why do you want to end the dictatorship?
 
-Because I think that this would make China, and the world:
+Because I think that this would make China and the world:
 
 * <<richer,richer>>
 * less likely to get into <<war,WW3>>
 
 [[richer]]
 ==== Why would democracy and freedom of speech make China richer?
 
-There is infinite debate about this out there, some examples:
+There is an infinite debate about this out there, some examples:
 
 * http://www.becker-posner-blog.com/2011/05/can-poor-countries-afford-democracy-becker.html
 * https://www.quora.com/Is-democracy-or-authoritarianism-better-for-developing-countries
 
 For:
 
-* dictatorships are more likely to start <<war>> or other crazy policies like the Great Leap Forward, which completely destroy the economy in one go
+* Dictatorships are more likely to start <<war,wars>> or other crazy policies like the Great Leap Forward, which completely destroy the economy in one go
```

Comment:
Is there a source for this choice: https://english.stackexchange.com/questions/9752/capitalization-for-a-bullet-list ?